### PR TITLE
[DOC] Small code and spelling fixes 

### PR DESCRIPTION
**Pull request by Tillsten**

### CODE REVIEW

File: docs/source/basis/members.rst [hunk 1]
```diff
@@ -69,7 +69,7 @@ To summarize:
     a [label="A value was previously set?"];
     a->b[label="No"];
```
Comment (MatthieuDartiailh) [inline docs/source/basis/members.rst:70]: This should be "Yes"

File: docs/source/basis/members.rst [hunk 2]
```diff
@@ -69,7 +69,7 @@ To summarize:
...
     a->c[label="Yes"];
```
Comment (MatthieuDartiailh) [inline docs/source/basis/members.rst:71]: And this "No"

Comment (MatthieuDartiailh) [reply docs/source/basis/members.rst:71]: Later on the steps f and g have been inverted.

**Review by MatthieuDartiailh — CHANGES_REQUESTED** [review]
Thanks for your contribution ! I just noticed that the diagram for get still contains a couple of mistake. Could you fix them ?

File: docs/source/basis/members.rst (hunk 2 above)
Comment (Tillsten) [reply docs/source/basis/members.rst]: Done

**Review by MatthieuDartiailh — APPROVED** [review]

**Comment by MatthieuDartiailh** [comment]
Thanks !

**Pull request merged**